global-files/.github/labels.yml

```diff
@@ -166,4 +166,3 @@
 - name: "v3-migration 🤖"
   description: "Preparing for Exercism v3"
   color: "e99695"
-
```

Comment:
Could you verify that removing this line does not affect the generated `labels.yml` file? IIRC we had this empty line in here specifically for that reason.